setup/README.md

```diff
@@ -30,7 +30,16 @@ The second option could be useful for master node initialization.
 Or with one-command run:
 
 With Liqo peering:
-``` wget -O - https://raw.githubusercontent.com/netgroup-polito/edge-infrastructure-ansible/main/setup/edge-pc-local-setup.sh <remote_target_ip> <remote_target_user> <remote_target_password> | sudo bash ```
+```bash
```

Comment:
This information looks very similar to the one contained in the main setup.md file (in the root folder).
We should avoid duplicated information, which is hard to maintain.
In case we really need a second file here, it should be properly justified.